### PR TITLE
Form 21-8940: Ability to populate Recent Education and Training questions#14540

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Form 21-8940: Ability to populate Recent Education and Training questions

As a Veteran submitting a claim for Individual Unemployability on the Vets.gov platform, I need the ability to populate the Recent Education and Training, so that I can provide additional evidence associated to my Individual Unemployability while completing the Form 21-8940 application interview.

## Testing done
- [x] local testing
- [x] e2e tests

## Screenshots
screen shot 2018-12-11 at 12 29 51 pm

## Acceptance criteria
- [x] The user is able to populate the following: 
- [x] In the time since you became too disabled to work, have you had any education or training? { radio }
- [x] Yes
- [x] No
- [x] If Yes, Type of education or training { text box }
- [x] From { date picker }
- [x] To { date picker }
- [x] The user is able to add another
- [x] The user is able to move to the next step in the process, Story#14541: Form 21-8940: Ability to populate Additional Information questions
- [x] Field validations (if applicable per the linked document for Validations above) are enforced in the UI
- [x] Error messages (if applicable per the linked document for Validations above) are displayed in the UI

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/components/EducationTrainingField.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  let from = '';
+  let to = '';
+  if (dates) {
+    from = formatReviewDate(dates.from);
+    to = formatReviewDate(dates.to);
+  }
+
+  return (
+    <div>
+      <strong>{name}</strong>
```
Comment (aaronjlech) [inline src/applications/disability-benefits/all-claims/components/EducationTrainingField.jsx]: Do we need to conditional check this if it's empty so it doesn't show `undefined` or will it show blank?

**Review by aaronjlech — COMMENTED** [review]
Looks good, just one little question

File: src/applications/disability-benefits/all-claims/components/EducationTrainingField.jsx (hunk 1 above)
Comment (zurbergram) [reply src/applications/disability-benefits/all-claims/components/EducationTrainingField.jsx]: Shows blank

**Review by aaronjlech — APPROVED** [review]
looks good!

**Review by jbalboni — APPROVED** [review]

**Pull request merged**